### DISCUSSION TITLE
The IP Allow/Deny configuration file

### DISCUSSION THREAD

**Discussion by Berwynnnn** [Security-Onion-Solutions/securityonion#10701] [Unsupported Versions] [answered]
I edited a file that allows me to add ip allow and ip deny for different protocols like example ssh, ftp, etc. but i cannot remember the name or location of the file. does anyone know where i can find the file again?

**Comment by petiepooo** [comment, marked as answer]
In /opt/so/saltstack/local/salt/firewall/

See https://docs.securityonion.net/en/latest/firewall.html

**Comment by InfosecGoon** [comment]
The easiest way to make these changes is with the so-firewall tool.

More information here: https://docs.securityonion.net/en/2.3/firewall.html

**Comment by Berwynnnn** [comment]
Thank you for the replies.
